package.json

```diff
@@ -1,37 +1,50 @@
 {
   "name": "@open-formulieren/formio-renderer",
-  "version": "1.0.0",
+  "version": "0.1.0",
   "description": "Alternative Form.io renderer made for Open Forms.",
-  "author": "",
-  "license": "EUPL-1.2",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/open-formulieren/formio-renderer.git"
-  },
-  "main": "dist/index.js",
-  "module": "dist/index.modern.js",
+  "main": "./dist/cjs/index.js",
+  "module": "./dist/esm/index.js",
+  "types": "./dist/esm/index.d.ts",
   "source": "src/index.tsx",
+  "exports": {
+    ".": {
+      "import": "./dist/esm/index.js",
+      "require": "./dist/cjs/index.js",
+      "default": "./dist/cjs/index.js"
+    },
+    "./esm/": "./dist/esm/"
+  },
   "scripts": {
-    "build": "microbundle-crl --no-compress --format modern,cjs",
-    "start": "microbundle-crl watch --no-compress --format modern,cjs",
-    "test": "run-s test:unit test:lint test:build",
-    "test:build": "run-s build",
-    "test:lint": "eslint .",
-    "test:unit": "cross-env CI=1 react-scripts test --env=jsdom",
```

Comment:
I removed this because there are no tests as of yet and we plan to rely on storybook test runner with the interaction tests anyway.